### PR TITLE
Added support for qualified event name.

### PR DESCRIPTION
**Pull request by milkovsky**

### CODE REVIEW

**Comment by milkovsky** [comment]
I added getQualifiedEventName() and try to trigger event both by the generic and qualified names.

File: src/Event/EntityEvent.php [hunk 1]
```diff
@@ -16,4 +16,23 @@
...
 class EntityEvent extends GenericEvent {
 
+  /**
+   * Provides qualified event name.
+   *
+   * More specific name for entity event. Adds bundle to the event name.
+   *
+   * @param $event_name
+   *   Event name.
+   *   @todo Unnecessary parameter. We could use $this->getName() but it is
```
Comment (fago) [inline src/Event/EntityEvent.php:26]: Fix or remove.

**Comment by milkovsky** [comment]
The PR is outdated due to https://www.drupal.org/node/2658842#comment-10819272 changes

**Comment by milkovsky** [comment]
Moved to https://github.com/fago/rules/pull/389

**Pull request closed without merging**